src/main/java/pro/beam/api/resource/chat/methods/WhisperMethod.java

```diff
@@ -0,0 +1,46 @@
+package pro.beam.api.resource.chat.methods;
+
+import com.google.common.collect.ImmutableList;
+import pro.beam.api.resource.BeamUser;
+import pro.beam.api.resource.chat.AbstractChatMethod;
+
+import java.util.List;
+import java.util.Random;
+
+public class WhisperMethod extends AbstractChatMethod {
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        protected BeamUser to;
+        protected String message;
+
+        public Builder to(BeamUser u) {
+            this.to = u;
+            return this;
+        }
+
+        public Builder send(String message) {
+            this.message = message;
+            return this;
+        }
+
+        public WhisperMethod build(int id) {
+            WhisperMethod method = new WhisperMethod();
+            method.arguments = ImmutableList.of(this.to.username, this.message);
+            method.id = id;
+
+            return method;
+        }
+
+        public WhisperMethod build() {
+            return this.build(new Random().nextInt());
```

Comment:
Good catch. I added a few constructors to AbstractChatMethod so the ID can be set, or it falls back to the random number generator. In a future PR, I'll update the RNG to be a sequential one.